osbs/utils.py

```diff
@@ -233,7 +234,9 @@ def checkout_git_repo(git_uri, git_ref, git_branch=None, max_retries=GIT_MAX_RET
     # git will remove the empty directory (why?!)
     args = ['git', 'clone', git_uri]
     if git_branch:
-        args += ['-b', git_branch]
+        args += ['-b', git_branch, '--single-branch']
+    if depth > 0:
```

Comment:
If depth assumes its default value (None), it will fail here in Python 3.

Reply:
Fixed.